packages/yarnpkg-pnpify/sources/sdks/vscode.ts

```diff
@@ -108,6 +102,24 @@ export const generateSvelteLanguageServerWrapper: GenerateIntegrationWrapper = a
   });
 };
 
+export const generateDefaultWrapper: GenerateDefaultWrapper = async (pnpApi: PnpApi) => {
+  await addVSCodeWorkspaceConfiguration(pnpApi, VSCodeConfiguration.settings, {
+    [`search.exclude`]: {
+      [`**/.yarn`]: true,
+      [`**/.pnp.*`]: true,
+    },
+  });
+
+  await addVSCodeWorkspaceConfiguration(pnpApi, VSCodeConfiguration.extensions, {
+    [`recommendations`]: [
+      `arcanis.vscode-zipfs`,
+    ],
+  });
+};
+
+// eslint-disable-next-line arca/no-default-export
+export default generateDefaultWrapper as DefaultSdk;
```

Comment:
I prefer to avoid default exports - can you turn it into a named one?

Reply:
Sure!

Reply:
Or even better, why not put it directly in `VSCODE_SDKS` with `null` for the package name?

Reply:
Good idea! Done 👍